ext/js/data/options-util.js

```diff
@@ -1389,6 +1390,26 @@ export class OptionsUtil {
         }
     }
 
+    /**
+     * - Rename `selection-text` to `popup-selection-text`
```

Comment:
```suggestion
     * - Renamed `selection-text` to `popup-selection-text`.
```
(toasted-nutbread liked to have these comments in past tense with punctuation)